src/Compilers/Core/Portable/Symbols/WellKnownMemberNames.cs

```diff
@@ -343,7 +343,7 @@ public static class WellKnownMemberNames
         /// </summary>
         public const string SliceMethodName = "Slice";
 
-        // PROTOTYPE: internal since this has yet to be approved
-        internal const string CloneMethodName = "Clone";
+        // internal until we settle on this long-term
+        internal const string CloneMethodName = "<>Clone";
```

Comment:
For other unspeakable names, don't we use a key letter (such as "k" for backing fields)?
```
        internal static string MakeBackingFieldName(string propertyName)
        {
            Debug.Assert((char)GeneratedNameKind.AutoPropertyBackingField == 'k');
            return "<" + propertyName + ">k__BackingField";
        }
```

Reply:
Yeah, but those examples are generated names where there's a user-dependent component. This is always the same name, so I didn't think it was particularly hard to reverse engineer if we needed to.